src/mixins/hooksRegisterer.js

```diff
@@ -0,0 +1,61 @@
+import { arrayEach } from './../helpers/array';
+import { defineGetter, objectEach } from './../helpers/object';
+
+const MIXIN_NAME = 'hooksRegisterer';
+
+/**
+ * Mixin object to extend objects functionality for auto registering hooks in an Handsontable instance.
+ *
+ * @type {Object}
+ */
+const hooksRegisterer = {
```

Comment:
Maybe to make mixin name more understandable it could be renamed to `hooksRefRegisterer` or something similar to "ref" (as "reference"). As the purpose of the mixin is only to collect callback references to make them easily remove when not used anymore?